packages/lesswrong/components/votes/NamesAttachedReactionsVoteOnComment.tsx

```diff
@@ -724,10 +724,17 @@ export const AddReactionButton = ({voteProps, classes}: {
     <span
       ref={buttonRef}
       onClick={ev => setOpen(true)}
-      className={classes.addReactionButton}
+      className={classNames(classes.addReactionButton, "react-hover-style")}
     >
-      <InsertEmoticonOutlined/>
-  
+      <svg className={"MuiSvgIcon-root"} fill="none" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
svg file?

Reply:
Yeah fair. This was how EA forum had done it but sure seems like a good refactor time.